src/coprocessor/dag/executor/scan.rs

```diff
@@ -186,9 +176,7 @@ impl<C: ExecSummaryCollector, S: Store, T: InnerExecutor> Executor for ScanExecu
         }
     }
 
-    fn collect_execution_summaries(&mut self, target: &mut [ExecSummary]) {
-        self.summary_collector.collect_into(target);
-    }
+    fn collect_execution_summaries(&mut self, _: &mut [ExecSummary]) {}
```

Comment:
maybe we can provide a default impl for this function (for normal executor).

Reply:
All other impls forward to call `self.inner.collect_execution_summaries(target)`, but for this to be defaultable the Executor trait would need to know there may be an `inner` 🤔